usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -44,7 +53,7 @@ def search_transactions(filters, fields, sort, order, lower_limit, limit):
     '''
     keyword = filters['keyword']
     query_fields = [TRANSACTIONS_LOOKUP[i] for i in fields]
-    query_fields.extend(['piid', 'fain'])
+    query_fields.extend(['piid', 'fain', 'uri', 'display_award_id'])
```

Comment:
I ran the latest `es_loader` for 2018 and don't have `display_award_id` in any documents or in the index mappings.

Reply:
That is not an issue with this code. I can show you the new indices with `display_award_id` in the documents and help you figure out the loader script